Tests/modules/network_related/test__ssl.py

```diff
@@ -15,7 +15,7 @@
 from iptest import IronPythonTestCase, is_cli, is_netcoreapp, retryOnFailure, run_test, skipUnlessIronPython
 
 SSL_URL      = "www.python.org"
-SSL_ISSUER   = "CN=GlobalSign Atlas R3 DV TLS CA 2022 Q3, O=GlobalSign nv-sa, C=BE"
```

Comment:
Is this change really needed for the _winapi stuff?

Reply:
nm, I see now

Reply:
I just wanted the green checkmark so I snuck it in. :smile: